Makefile

```diff
@@ -0,0 +1,22 @@
+lint:
+	npm run lint
+
+test:
+	npm run test
+
+end-to-end-test:
+	git clone https://github.com/weldr/welder-deployment
+	make -C welder-deployment/ weld-f25
+
+	sudo docker build -t weld/end-to-end:latest .
+
+	make -C welder-deployment/ repos
+	sudo docker-compose build -f welder-deployment/docker-compose.yml
+	wget https://s3.amazonaws.com/atodorov/metadata_centos7.db.gz -O welder-deployment/metadata.db.gz
+	gunzip ./welder-deployment/metadata.db.gz
+	make -C welder-deployment/ import-metadata
+	sudo docker-compose -p welder up -d -f welder-deployment/docker-compose.yml
```

Comment:
Should be `sudo docker-compose -f welder-deployment/docker-compose.yml  -p welder up -d`